### PR TITLE
moving sp1k4 and im5k4 camera sync into daq

### PR DESCRIPTION
**Pull request by tongju12**
James wants to move IM5K4 and SP1K4 camera sync PV into daq in alarm system

### CODE REVIEW

File: Spreadsheet/KFE/TMO-alarms.csv [hunk 1]
```diff
@@ -80,13 +78,16 @@
...
 ,,pva://DAQ:NEH:tmo:0:Damage_hsd,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_bld,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_epics,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_atm,,,,,
+1,SP1K4,,,,,,
+,,SP1K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,
+,,SP1K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,
+1,IM5K4
+,,IM5K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,
+,,IM5K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,
```
Comment (ZLLentz) [inline Spreadsheet/KFE/TMO-alarms.csv:93]: Each line should have exactly 7 commas or the parser breaks
```suggestion
,,SP1K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,,
,,SP1K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,,
1,IM5K4,,,,,,
,,IM5K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,,
,,IM5K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,,
```

Comment (tongju12) [reply Spreadsheet/KFE/TMO-alarms.csv]: Thanks Zach. I fix it. It is strange that all previous PV has less comma. But I fixed those according to your suggestion.

File: Spreadsheet/KFE/TMO-alarms.csv [hunk 2]
```diff
@@ -80,13 +78,16 @@
...
 ,,pva://DAQ:NEH:tmo:0:Damage_hsd,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_bld,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_epics,,,,,
 ,,pva://DAQ:NEH:tmo:0:Damage_atm,,,,,
+1,SP1K4,,,,,,
+,,SP1K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,,
+,,SP1K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,,
+1,IM5K4,,,,,,
+,,IM5K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,,
+,,IM5K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,,
```
Comment (ZLLentz) [inline Spreadsheet/KFE/TMO-alarms.csv:93]: I miscounted, only the IM5K4 line needed an adjustment previously
```suggestion
,,SP1K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,
,,SP1K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,
1,IM5K4,,,,,,
,,IM5K4:PPM:CAM:TSS:SyncStatus,Camera Sync Status,,,,
,,IM5K4:PPM:CAM:TSS:DiffVsIntAvg,Sync Diff,,,,

```

Comment (tongju12) [reply Spreadsheet/KFE/TMO-alarms.csv]: I updated again, Zach.

**Comment by ZLLentz** [comment]
All checks passed, marking as ready and merging!

**Pull request merged**